exporters/autoexport/go.mod

```diff
@@ -27,7 +27,9 @@ require (
 	golang.org/x/net v0.10.0 // indirect
 	golang.org/x/sys v0.8.0 // indirect
 	golang.org/x/text v0.9.0 // indirect
-	google.golang.org/genproto v0.0.0-20230306155012-7f2fa6fef1f4 // indirect
+	google.golang.org/genproto v0.0.0-20230530153820-e85fd2cbaebc // indirect
+	google.golang.org/genproto/googleapis/api v0.0.0-20230526203410-71b5a4ffd15e // indirect
```

Comment:
Why not `20230530153820-e85fd2cbaebc` like in the line above?

Reply:
🤷🏻 this is what `go get` had generated

Reply:
Updated it to match